deploy/cephfs/kubernetes/csi-cephfsplugin.yaml

```diff
@@ -90,6 +89,21 @@ spec:
               mountPath: /dev
             - name: ceph-csi-config
               mountPath: /etc/ceph-csi-config/
+        - name: liveness-prometheus
+          image: quay.io/cephcsi/cephcsi:canary
```

Comment:
can you follow the order like other containers

* image
* args
* env
* volumeMounts